pkg/virt-handler/container-disk/mount.go

```diff
@@ -189,6 +201,7 @@ func (m *mounter) setMountTargetRecord(vmi *v1.VirtualMachineInstance, record *v
 	return nil
 }
 
+
```

Comment:
Is this a redundant newline?

Reply:
Yes, but this is deleted in future commits. I can delete it in this commit as well